lib/ASRouter.jsm

```diff
@@ -2095,6 +2185,9 @@ class _ASRouter {
           action.data.value
         );
         break;
+      case "SET_GROUP_STATE":
```

Comment:
Can we also add a user action to block a group? It should apply to future messages in the group as well.